### ISSUE TITLE
Find a way to remove delay before using serial

### ISSUE THREAD

**Issue by dylif** [dylif/humid-esp32-pio#3]
Right now, the app delays for several seconds before starting to allow the serial monitor to connect. It would be better to use a true UART peripheral to avoid this.

**Comment by dylif** [comment]
It seems like removing this delay would not be possible without using the TX/RX pins on the chip, which are not easily accessible through the case. Marking as won't fix since this is out of scope and not really a big issue.
